src/style.css

```diff
@@ -0,0 +1,157 @@
+* {
+  margin: 0;
+}
+
+/* .card__info-product {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--mainColor);
+  margin: 0 0 4px;
+} */
+
+/* p {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--mainColor);
+  margin: 0;
+} */
+
+:root {
+  --mainColor: #060b35;
+  --blueAccent: #00acdc;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  box-sizing: border-box;
+}
+
+.card--size {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__logo {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-name {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--mainColor);
+  margin: 0 0 4px;
+}
+
+.product-code {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 0 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+.card__stars-section {
+  display: flex;
+}
+
+.card__stars-star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.card__stars-star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__stars-section--star-0 :nth-child(-n+0),
+.card__stars-section--star-1 :nth-child(-n+1),
+.card__stars-section--star-2 :nth-child(-n+2),
+.card__stars-section--star-3 :nth-child(-n+3),
+.card__stars-section--star-4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.card__stars-reviews {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__info {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  color: var(--mainColor);
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  margin: auto;
```

Comment:
The better practice is to set the margin to 0 auto; when you want to centralize the element.